docs/explaining_ndcube/coordinates.rst

```diff
@@ -225,7 +225,7 @@ The physical types of extra coordinates are also returned by `~ndcube.NDCube.arr
 .. code-block:: python
 
   >>> my_cube.array_axis_physical_types
-  [('custom:pos.helioprojective.lat', 'custom:pos.helioprojective.lon', 'time'), ('custom:pos.helioprojective.lat', 'custom:pos.helioprojective.lon'), ('em.wl',)]
```

Comment:
This is our code, so I think rather than doing this we should just fix the code here:

https://github.com/sunpy/ndcube/blob/62bb3693046470b2db1ce39f7241144973789a8f/ndcube/ndcube.py#L428-L429

Reply:
Oh it's code in ndcube, I assumed it was astropy. 

I can change the code here instead then.